scss/_print.scss

```diff
@@ -18,18 +24,37 @@
     text-decoration: underline;
   }
 
+  a[href]::after {
+    content: " (" attr(href) ")";
+  }
+
   abbr[title]::after {
     content: " (" attr(title) ")";
   }
 
+  //
+  // Don't show links that are fragment identifiers,
+  // or use the `javascript:` pseudo protocol
+  //
+
+  a[href^="#"]::after,
+  a[href^="javascript:"]::after {
+    content: "";
+  }
+
   pre,
   blockquote {
-    border: $border-width solid #999;
```

Comment:
\*shrug* I don't know the context for this one off the top of my head. Seems like a reasonable deviation from upstream though.